README.md

```diff
@@ -6,16 +6,16 @@ A set of MSBuild tasks and targets to automatically update xliff (.xlf) files fo
 
 |Windows x64 |
 |:------:|
-|[![][win-x64-build-badge]][win-x64-build]|
+|[![Build Status][win-x64-build-badge]][win-x64-build]|
```

Comment:
[![Build Status](https://devdiv.visualstudio.com/DevDiv/_apis/build/status/DotNet/xliff-tasks/master/dotnet.xliff-tasks?branchName=master)](https://devdiv.visualstudio.com/DevDiv/_build/latest?definitionId=11329&branchName=master) ?

Reply:
I prefer to use "footnote style" linking to make the Markdown easier to read. Though "win-x64" may no longer be the best way to refer to the build...

Reply:
I guess the problem is its pointing to the wrong build definition?

Reply:
What should it be pointing to?

Reply:
presumably [this](https://devdiv.visualstudio.com/DevDiv/_build?definitionId=11329&_a=summary) definition

Reply:
Isn't that the one I'm pointing it to? The link I've used is a little different--it's the one Azure DevOps gave me specifically for the badge--but its still going to definition 11329.

Reply:
my mistake I mis-interpreted the diff.